.github/workflows/workflow.yml

```diff
@@ -13,12 +13,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node-version: [8.17.0, 15.x]
+        node-version: [10.x, 16.x]
```

Comment:
We should use `10.0.0` instead of the latest version of `10.*.*` to cover more users, what do you think?